modules/db/src/main/resources/db/migration/V0001__init.sql

```diff
@@ -6,7 +6,7 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
-CREATE TYPE title AS ENUM ('GM', 'WGM', 'IM', 'WIM', 'FM', 'WFM', 'CM', 'WCM', 'NM', 'WNM');
```

Comment:
We shouldn't change the migration scripts. Because it'll mess with the migration process. For now, I think the best way is keep database schema as it is and create an issue to change this later.

I think I'll combine these schemas right before We deploy it to prod.